File: C++/IV - Programming Exercise (Distance Traveled).cpp
#include<iostream>

using namespace std;
// Display
int main()
{ 
    string name;
    int speed,hours;
    cout << "Full Name: " << endl;
    getline(cin, name);
    cout << "What is the speed of the vehicle in mph? " << endl;
    cin >> speed;
    cout << "How many hours has it traveled? " << endl;
    cin >> hours;
    
    cout << ("Hours\t\tDistance Traveled") << endl;
// Loop 
    for (int i = 1; i <= hours; i++){
        cout << i << "\t\t" << i * speed << endl;
    }
    
    return 0;
}

File: Python/IV - Programming Exercise (Distance Traveled).py
# Display
str_name = str(input("Full Name: "))
speed = int(input("What is the speed of the vehicle in mph? "))
hours = int(input("How many hours has it traveled? "))

# Loop & Display
distance = 0
output = "\nHour       Distance traveled\n" \
             "-------------------------\n"

for hour in range(hours):
    distance = speed * (hour + 1)
    
    output += format(hour + 1) + "     " + \
              format(distance) + "\n"
print(output)
